introduction.md

```diff
@@ -1,5 +1,4 @@
-Hi my name is Alex and my favorite path is JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?